src/sentry/static/sentry/app/views/settings/organizationMembers/organizationMemberDetail.tsx

```diff
@@ -0,0 +1,421 @@
+                        >
+                          {t('Resend Invite')}
+                        </Button>
+                      )}
+                    </InviteActions>
+                  </InviteSection>
+                )}
+              </OverflowWrapper>
+            </PanelItem>
+          </PanelBody>
+        </Panel>
+
+        {showAuth && (
+          <Panel>
+            <PanelHeader>{t('Authentication')}</PanelHeader>
+            <PanelBody>
+              <Field
+                alignRight
+                flexibleControlStateSize
+                label={t('Reset two-factor authentication')}
+                help={t(
+                  'Resetting two-factor authentication will remove all two-factor authentication methods for this member.'
+                )}
+              >
+                <Tooltip
+                  data-test-id="reset-2fa-tooltip"
+                  disabled={this.showResetButton()}
+                  title={this.getTooltip()}
+                >
+                  <Confirm
+                    disabled={!this.showResetButton()}
+                    message={tct(
+                      'Are you sure you want to disable all two-factor authentication methods for [name]?',
+                      {name: member.name ? member.name : 'this member'}
+                    )}
+                    onConfirm={this.handle2faReset}
+                    data-test-id="reset-2fa-confirm"
+                  >
+                    <Button data-test-id="reset-2fa" priority="danger">
+                      {t('Reset two-factor authentication')}
+                    </Button>
+                  </Confirm>
+                </Tooltip>
+              </Field>
+            </PanelBody>
+          </Panel>
+        )}
+
+        <RoleSelect
+          enforceAllowed={false}
+          disabled={!canEdit}
+          roleList={member.roles}
+          selectedRole={member.role}
+          setRole={slug => this.setState({member: {...member, role: slug}})}
+        />
+
+        <TeamSelect
+          organization={organization}
+          selectedTeams={member.teams}
+          disabled={!canEdit}
+          onAddTeam={this.handleAddTeam}
+          onRemoveTeam={this.handleRemoveTeam}
+        />
+
+        <Footer>
+          <Button
+            priority="primary"
+            busy={this.state.busy}
+            onClick={this.handleSave}
+            disabled={!canEdit}
+          >
+            {t('Save Member')}
+          </Button>
+        </Footer>
+      </React.Fragment>
+    );
+  }
+}
+
+export default withOrganization(OrganizationMemberDetail);
+
+const ExtraHeaderText = styled('div')`
+  color: ${p => p.theme.gray2};
+  font-weight: normal;
+  font-size: ${p => p.theme.fontSizeLarge};
+`;
+
+const Details = styled('div')`
+  display: grid;
+  grid-auto-flow: column;
+  grid-template-columns: 2fr 1fr 1fr;
+  grid-gap: ${space(2)};
+  width: 100%;
+
+  ${p => `
+  @media (max-width: ${p.theme.breakpoints[0]}) {
```

Comment:
I've typically just interpolated for the breakpoint, not the whole media query body